install/kubernetes/helm/istio-remote/charts/security/templates/deployment.yaml

```diff
@@ -29,13 +31,18 @@ spec:
             - --grpc-port=8060
             - --grpc-hostname=citadel
             - --citadel-storage-namespace={{ .Release.Namespace }}
+          {{- if .Values.selfSigned }}
+            - --self-signed-ca=true
```

Comment:
Does the two cluster can communicate with each other if using different root cert?

Reply:
@gyliu513 this looks like a copy and paste from the main helm chart.  I think that is fine for now.

Reply:
yeah, what Steve said. The security and sidecar injector charts are exact copies of the subcharts in `install/kubernetes/helm/istio`

Reply:
But my question is does mTLS works cross clusters for such case?

Reply:
@gyliu513  -- AFAIK, yes, as long as each cluster's root cert was signed by the same CA.